binding/go/porcupine_test.go

```diff
@@ -13,28 +13,61 @@ package porcupine
 import (
 	"encoding/binary"
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"math"
 	"os"
 	"path/filepath"
 	"testing"
+	"reflect"
 )
 
-var pvTestAccessKey string
+var (
+	pvTestAccessKey string
+	p               Porcupine	
+)
 
 func TestMain(m *testing.M) {
+
 	flag.StringVar(&pvTestAccessKey, "access_key", "", "AccessKey for testing")
 	flag.Parse()
+
 	os.Exit(m.Run())
 }
 
-func TestProcess(t *testing.T) {
+func appendLanguage(s string, language string) string {
+	if language == "en" {
+		return s;
+	}
+	return s + "_" + language
+}
+
+func getTestModelPath(language string) string {
+	modelPath, _ := filepath.Abs(fmt.Sprintf(
```

Comment:
with nested calls that don't fit into a line please prefer to break them like below:

```go
relPath = fmt.Sprintf(
		"../../lib/common/%s.pv",
		appendLanguage("porcupine_params", language))
filepath.Abs(relPath)
```